js/src/modals/Shapeshift/AwaitingDepositStep/awaitingDepositStep.js

```diff
@@ -79,4 +79,24 @@ export default class AwaitingDepositStep extends Component {
       </div>
     );
   }
+
+  renderAddress (depositAddress) {
+    return (
+      <div className={ styles.addressInfo }>
+        <a
+          href={ `bitcoin:${depositAddress}` }
+          target='_blank'
+        >
+          <QrCode
+            className={ styles.qrcode }
+            value={ depositAddress }
```

Comment:
Don't we want to use the `bitcoin:${despositAddress}` format for BTC exchanges ? It feels unbalanced, but aren't QR code scanners on mobile suppose to trigger BTC app if available ?